packages/shopping/src/models/envelope.model.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback4-example-shopping
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Entity, model, property} from '@loopback/repository';
+
+@model()
+export class Envelope extends Entity {
+  @property({
+    type: 'string',
+  })
+  from: string;
+
+  @property.apply({
```

Comment:
Could you clarify what `@property.apply()` is used for? I can't seem to find it in the LB4 docs.

Reply:
Weird. I think it is IDE auto complete.